test/parallel/test-vm-timeout.js

```diff
@@ -6,6 +6,9 @@ const vm = require('vm');
 // Test 1: Timeout of 100ms executing endless loop
 assert.throws(function() {
   vm.runInThisContext('while(true) {}', { timeout: 100 });
+}, function(err) {
```

Comment:
I think using a regexp as the second argument would be more succinct: `/^Error: Script execution timed out\.$/`